lib/shopify_graphql/client.rb

```diff
@@ -30,6 +30,7 @@ def client
     end
 
     def execute(query, headers: nil, **variables)
+      variables = convert_variables_to_camel_case(variables) if ShopifyGraphql.configuration.convert_case
```

Comment:
This doesn't really need to be conditional as if the variables are passed in `camelCase` they will stay that way.